delete_all_packages.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# This script allows you to mass delete all packages from a 'user' repository.
+# This only deletes 30 packages at a time. Repeat it until everything is deleted.
+
+import requests
+
+# Replace these variables with a Personal access token from GitHub
+your_token = 'ghp_'
+user = 'user'
+
+def delete_packages(pat, user):
+    headers = {
+        'Authorization': f'Bearer {pat}',
+        'X-GitHub-Api-Version': '2022-11-28',
+        'Accept': 'application/vnd.github+json'  # Required header for package deletion API
+    }
+
+    # Fetching package versions
+    response = requests.get(f'https://api.github.com/users/{user}/packages?package_type=nuget', headers=headers)
+    if response.status_code != 200:
+        print(f"Failed to fetch packages: {response.status_code}")
+        return
```

Comment:
Pretty sure you can replace that with

```suggestion
    response.raise_for_status()
```
to make it more robust. This will also set the exit code correctly when invoked as a script.